### DISCUSSION TITLE
Normalize Pivots

### DISCUSSION THREAD

**Discussion by overstartup** [DaveSkender/Stock.Indicators#982] [Help and support]
I need to have important areas by getting pivots and touch in that area around the pivot, i think it calls normalize.
It seems I can have all pivots by GetFractal or GetPivots. 

For example, i need to get the best point for these pivots:
20.09
20.06
20
19.92

Do you have any idea what is the best solution to have the pivot area instead of pivot point?

**Comment by DaveSkender** [comment]
Can you define “area” again?  I’m not understanding it.
